data.json

```diff
@@ -1305,5 +1300,11 @@
     "githubUsername": "Mo-Shakib",
     "imageUrl": "https://user-images.githubusercontent.com/50780268/131604251-f07f6811-d795-448f-a7b8-0f13b58c602f.png",
     "issueNumber": 675
-  }
+  },
+      { "name": "Komal Pal",
```

Comment:
Please can this be formatted @Komal7209 , so it's something like the example: 
![image](https://user-images.githubusercontent.com/70807500/131862320-317889af-f1bb-4d36-8e01-a2804688de00.png)

After, I think this can be merged! Thanks